QSB/OrbSync/OrbPatches.cs

```diff
@@ -1,11 +1,13 @@
-﻿using QSB.Events;
+﻿using QSB.EventsCore;
 using QSB.WorldSync;
 using UnityEngine;
 
 namespace QSB.OrbSync
 {
-    public static class OrbPatches
+    public class OrbPatches : QSBPatch
```

Comment:
i thought patch classes also had to be static for Harmony to work. if this works, it's great!

Reply:
i think only patch methods have to be static